CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v1.1.4 - 2022-07-13
+
+### Adds
+
+- Add format_nongpl extra to jsonschema install. This ensures draft7 format checkers validate format adherence as expected while also ensuring GPL-Licenced transitive dependencies are not installed.
+
+### Changes
+
+- Update jsonschema schema version dependency so that only versions greater than 4.4 are supported.
```

Comment:
```suggestion
- Update jsonschema dependency so that only versions greater than 4.7 are supported.
```

Reply:
BTW, what was the reason/benefit to end users of updating the minimum jsonschema version? Better validation? Bug fixes? Would be good to spell that out here.

Reply:
Resolved per #139